docs/src/adoc/index.adoc

```diff
@@ -597,6 +597,44 @@ LocalDate releaseDate = handle.createQuery(
     .findOnly();
 ----
 
+==== LIKE clauses
+
+Due to a built-in safety mechanism, `LIKE` clauses and argument binding
+don't work together intuitively and must be written a particular way. The following
+is what users would intuitively try to do:
+
+[source,java]
+----
+handle.createQuery("select name from things
+where name like '%:search%'")
+    .bind("search", "foo")
+    .mapTo(String.class)
+    .list()
+----
+
+Firstly, this query would try to select `where name like '%:search%'`
+_literally_, without binding any arguments. This is because proper `SqlParser`
+implementations do not bind arguments _inside string literals_.
```

Comment:
This isn't a `SqlParser` limitation, this is a consequence of how jdbc works.  We could change the parser any which way we want but the driver will not support doing this.

Reply:
We do actually specifically write our parsers to not even attempt it regardless of db capability. The text doesn't assign blame, it just says it will fail in sqlparsing already (to fail sooner than later) by design because jdbi is explicitly meant to do so, even if your maybe custom-made or patched db would support it.